todoman/interactive.py

```diff
@@ -530,7 +530,8 @@ def __init__(self, databases, formatter):
         Create a Main instance based on the Database objects that
         the regular Todoman cli module passes.
 
-        (Main, [Database], TodoFormatter) -> None
+        :type databases: list[Database]
```

Comment:
We still need to change this, because with the current model it doesn't make sense to pass a list of `Database`s.